gs-web-elasticsearch/src/main/java/mil/nga/giat/elasticsearch/ElasticConfigurationPage.java

```diff
@@ -330,7 +333,17 @@ protected Component getComponentForProperty(String id, IModel<ElasticAttribute>
                     } else {
                         Fragment f = new Fragment(id, "empty", ElasticConfigurationPage.this);
                         return f;
-                    }
+                    } 
+                } else if (property == ElasticAttributeProvider.ORDER) {
+                	TextField<Integer> order = new TextField<Integer>("order", new PropertyModel<Integer>(itemModel, "order"));

```

Comment:
Can you ensure you're using 4 spaces for indentation throughout the PR?

Reply:
Sure, I updated several of the files, replaced tabs with 4 spaces.  I'll update the remaining files shortly.

Reply:
All files that were changed under this branch have been updated, tabs were replaced by 4 spaces.  The only exception is gs-web-elasticsearch/src/main/java/mil/nga/giat/elasticsearch/ElasticConfigurationPage.html, which contained tab formatting prior to these updates.